src/pages/Login.jsx

```diff
@@ -60,26 +57,28 @@ class Login extends Component {
               type='email'
               placeholder='expamlple@gmail.com'
               className='login__container__row__input'
-              onChange={this.inputEmailChange}
+              onChange={(item) => this.inputChange(item.target.value, 'email')}
```

Comment:
`onChange={this.inputChange}` is enough, and add name attribute, after this just spread everything you need from event object in method 
`const { name, value } = event.target;`

Reply:
Thx for that, it's much more convenient.
```suggestion
    inputChange(event) {
    const { type, value } = event.target;
    this.setState({ [type]: value });
    ...
     <input type='password' className='login__container__row__input' onChange={this.inputChange} />
  }
```